src/System.Private.CoreLib/src/System/Runtime/InteropServices/ComEventsSink.cs

```diff
@@ -122,7 +122,7 @@ public ComEventsMethod AddMethod(int dispid)
 
         #endregion
 
-        private static Guid IID_IManagedObject = new Guid("{C3FCC19E-A970-11D2-8B5A-00A0C9B7C9C4}");
+        private static Guid IID_IManagedObject = new Guid(0xC3FCC19E, 0xA970, 0x11D2, 0x8B, 0x5A, 0, 0xA0, 0xC9, 0xB7, 0xC9, 0xC4);
```

Comment:
This check for `IID_IManagedObject` can be deleted. We do not have this interface in CoreCLR. We have deleted all places that dealt with it from unmanaged runtime, but missed this place.